distributor.php

```diff
@@ -183,6 +183,8 @@ function() {
 		) {
 			\Distributor\Connections::factory()->register( '\Distributor\InternalConnections\NetworkSiteConnection' );
 		}
+
+		load_plugin_textdomain( 'distributor', false, dirname( plugin_basename( __FILE__ ) ) . '/lang' );
```

Comment:
So this loads from the `lang` directory but that isn't a directory that we currently have in Distributor. I get that this is setting us up for the future but may be worth adding that directory as part of this PR?

Reply:
@dkotter We have that directory in the `stable` branch. We build the pot file and put it into `lang` folder for the release.